app/controllers/finance/provider/log_entries_controller.rb

```diff
@@ -6,7 +6,7 @@ class Finance::Provider::LogEntriesController < Finance::Provider::BaseControlle
   def index
     @search = ThreeScale::Search.new(params[:search] || params)
 
-    @log_entries = current_account.log_entries.scope_search(@search, false)
+    @log_entries = current_account.log_entries.scope_search(@search)
```

Comment:
what does this change?

Reply:
I noticed this method is unused in `scope_search` so we can just removed:
https://github.com/3scale/porta/blob/e1274be61b4ad0481d5ca25bd5da5d130acb86b2/app/lib/three_scale/search.rb#L102-L116

Reply:
Good. How about removing the method parameter as well?

Reply:
Yep, it's removed as well https://github.com/3scale/porta/pull/2896/files#diff-b775669694cafa0abb4123d16eba681f85992932ecc98eebb8c651b0384a2dbfR103